astropy/io/fits/tests/test_hdulist.py

```diff
@@ -620,8 +619,7 @@ def test_update_with_truncated_header(self):
         with fits.open(self.temp('temp.fits')) as hdul:
             assert (hdul[0].data == data).all()
 
-    @pytest.mark.xfail(platform.system() == 'Windows',
```

Comment:
I have no idea why two tests in FITS marked as xfail on Windows would start passing for this PR, causing Windows job to fail. 🤷‍♀